frontend/src/metabase/query_builder/components/chart-type-selector/ChartTypeSettings/ChartTypeSettings.tsx

```diff
@@ -0,0 +1,34 @@
+import { t } from "ttag";
+
+import { ChartTypeList, type ChartTypeListProps } from "../ChartTypeList";
+
+import { OptionLabel } from "./ChartTypeSettings.styled";
+
+type ChartTypeSettingsProps = {
```

Comment:
I wanted to make sure the type accurately reflected that the data is being drilled through this component so no point in duplicating the types